frontend/packages/kubevirt-plugin/src/selectors/immutable/template/combined.ts

```diff
@@ -138,3 +139,8 @@ export const getITemplateDefaultWorkload = (template: ITemplate): WorkloadProfil
   WorkloadProfile.getAll().find(
     (w) => iGetLabels(template).get(`${TEMPLATE_WORKLOAD_LABEL}/${w.getValue()}`) === 'true',
   );
+
+export const iGetTemplateGuestToolsDisk = (tmp: ITemplate, diskName: string = WINTOOLS_DISK_NAME) =>
```

Comment:
I think we should rather use isWinToolsImage function to have the detection consistent across the code base